tests/js/spec/views/organizationDiscover/discover.spec.jsx

```diff
@@ -651,8 +651,9 @@ describe('Discover', function() {
         expect.anything(),
         expect.objectContaining({
           data: expect.objectContaining({
-            start: '2017-10-02T22:41:20',
-            end: '2017-10-16T22:41:20',
+            start: '2017-10-03T02:41:20',
+            end: '2017-10-17T02:41:20',
+            utc: null,
```

Comment:
We're changing this test because previously the utc value was `true`. The UI behavior doesn't change because in `organizationDiscover/index` we update the store based on user's configuration, but this test only tests `organizationDiscover/discover` which has a default prop for `utc=true`